### PR TITLE
Configuration option for chunk type

### PR DESCRIPTION
**Pull request by danni-m**
- add module configuration for chunk type
- add chunk type to TS.INFO
- add debug flag to TS.INFO

### CODE REVIEW

**Comment by danni-m** [comment]
@gkorland @jeffreylovitz ping

File: src/module.c [hunk 1]
```diff
@@ -292,18 +292,27 @@ static int parseCountArgument(RedisModuleCtx *ctx,
...
 
-    if (argc != 2) {
+    if (argc < 2 || argc > 3) {
         return RedisModule_WrongArity(ctx);
     }
 
+    RedisModuleKey *key = RedisModule_OpenKey(ctx, argv[1], REDISMODULE_READ);
     Series *series;
-    RedisModuleKey *key;
-    const int status = GetSeries(ctx, argv[1], &key, &series, REDISMODULE_READ);
```
Comment (filipecosta90) [inline src/module.c:301]: @danni-m I believe we should keep the GetSeries wrapper. Any reason for changing back to OpenKey+KeyType+GetValue?

**Review by filipecosta90 — CHANGES_REQUESTED** [review]
LGTM @danni-m . Only one comment regarding the `GetSeries` usage.

File: src/module.c (hunk 1 above)
Comment (danni-m) [reply src/module.c:301]: @filipecosta90 I remember to have some reasoning behing it, but looking at the code I can't seem to understand why I did it. will revert back to using GetSeries

**Review by filipecosta90 — APPROVED** [review]

**Comment by K-Jo** [comment]
@danni-m doesn't look like any documentation was update? does it need to?  if so let's create an issue

**Pull request merged**